commitizen/cli.py

```diff
@@ -114,11 +114,13 @@ def __call__(
         "commands": [
             {
                 "name": ["init"],
+                "description": "init commitizen configuration",
                 "help": "init commitizen configuration",
```

Comment:
Should we remove `help`? is it useful at all?

Reply:
@marcosdotme Could you please check whether we could just remove "help"? This PR mostly looks good to me 👍

Reply:
I we remove `help`, the command is not shown when running `cz --help`

![image](https://github.com/commitizen-tools/commitizen/assets/48416792/7143f35d-6c39-47bc-82f8-bd611dc75ef6)

CC: @Lee-W

Reply:
Yep, I guess we'll have to keep them then. I'm not sure whether there's a way we can remove this duplication. But let's keep it for another PR.